api/hasura/actions/create_circle.ts

```diff
@@ -36,7 +37,8 @@ export default async function handler(req: VercelRequest, res: VercelResponse) {
     const ret: ValueTypes['create_circle_response'] =
       await gql.insertCircleWithAdmin(
         input,
-        process.env.COORDINAPE_USER_ADDRESS
+        req.body.session_variables['x-hasura-address'],
```

Comment:
I wrote a composable request body parser in #508. Can this be rebased on top of that with the session variable zod parser updated?

Reply:
it's here: https://github.com/coordinape/coordinape/blob/c3d60f85c48045820557ebe00d3dfcb751dcb4f7/src/lib/zod/index.ts#L42-L64

Reply:
Sure